src/scripts/main.js

```diff
@@ -2,7 +2,18 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const result = document.querySelector('table');
```

Comment:
make meaningful name
```suggestion
const table = document.querySelector('table');
```

Reply:
Done